javascripts/releases.js

```diff
@@ -0,0 +1,78 @@
+var ev3devRepoReleaseCacheKey = 'ev3dev-repo-release-cache';
+var releaseCacheTimeMillis = 1 * 24 * 60 * 60 * 1000;
+
+var releasePlatformRegexes = {
+    ev3: "ev3-[\\w\\d-]*\\.img\\.xz",
+    rpi: "rpi-[\\w\\d-]*\\.img\\.xz",
+    rpi2: "rpi2-[\\w\\d-]*\\.img\\.xz",
+    evb: "evb-[\\w\\d-]*\\.img\\.xz",
+}
+
+function supportsHtml5Storage() {
+    try {
+        return 'localStorage' in window && window['localStorage'] !== null;
```

Comment:
Does `localStorage` persist across browser restarts? what about CTRL+F5 refresh.

I think caching the value is a good idea, but when there is a new release, people could be stuck on the old release for 24 hours, which would be annoying.